src/typings/che.d.ts

```diff
@@ -29,10 +29,13 @@ declare namespace che {
     usedResources?: string;
   }
 
+  export type WorkspaceStorageType = 'async' | 'ephemeral' | 'persistent';
+
   export interface WorkspaceSettings {
     cheWorkspaceDevfileRegistryUrl?: string;
     cheWorkspacePluginRegistryUrl: string;
-    'che.workspace.persist_volumes.default': 'false' | 'true';
+    'che.workspace.storage.available_types': string;
```

Comment:
Why 'false' | 'true' ->  string ?

Reply:
Not sure I've understood your question correctly, please elaborate on it.

Reply:
I am sure that using type  'false' | 'true' is better in this case.
Could we using something else for 'che.workspace.persist_volumes.default' (instead of 'false' or 'true' ) ?

Reply:
I got what's confused you.

This field was removed:
```typescript
    'che.workspace.persist_volumes.default': 'false' | 'true';	
```

These two fields were added:
```typescript
    'che.workspace.storage.available_types': string;
    'che.workspace.storage.preferred_type': WorkspaceStorageType;
```

Reply:
I have got the difference:
'che.workspace.persist_volumes.default': -> 'che.workspace.storage.available_types'
Skiped it in the first time

Reply:
ok